docs/blog/tinyglobby-migration.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Tinyglobby migration and matching at the speed of light
+author:
+  - name: Madeline Gurriarán
+sidebar: false
+# placeholder
+date: 2025-05-22
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: Tinyglobby migration and matching at the speed of light
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/tinyglobby-migration
+  - - meta
+    - property: og:description
+      content: Our journey so far as a community leading into 2025
+  - - meta
+    - property: og:image
+      content: https://e18e.dev/og/tinyglobby-migration.png
+---
+
+_May XX, 2025_
+
+# ![Tinyglobby migration and matching at the speed of light)](/og/tinyglobby-migration.png)
+
+My story with dependencies is odd. I started coding in JavaScript about five years ago,
```

Comment:
i wonder if this should live in an `About me` section, then the first heading-less paragraph would just be a brief intro to what this post is

e.g.

```md
This is the story of the tinyglobby library - our journey here within the e18e community and what's next.
```